stories/jsx-pragma.tsx

```diff
@@ -0,0 +1,40 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+// TypeScript thinks we're using the automatic runtime however we've forced Babel
```

Comment:
How does babel handle `@jsxRuntime` exactly? I don't see any code references to it, and am unsure of where the magic comes from

Reply:
It forces the runtime to be whatever is declared (classic or runtime). It's a babel thing unfortunately not supported by TS. Useful for testing :)

Reply:
Cool, found it here: https://babeljs.io/docs/en/babel-plugin-transform-react-jsx

Reply:
Oh! Sorry should have gotten it for you.